### PR TITLE
Walkthrough 2

### PR DESCRIPTION
**Pull request by david-martin**

### CODE REVIEW

**Comment by dlabaj** [comment]
@david-martin Do you want this merged or are you still working on it?  Feel free to merge it if it's ready.

**Comment by david-martin** [comment]
@dlabaj still wip.
I suspect this will be in progress for a few days at least which working out issues.

**Comment by aidenkeating** [comment]
@david-martin Would you mind taking a look?

**Comment by david-martin** [comment]
@aidenkeating :eyes:

File: docs/modules/ROOT/pages/_partials/calling-fuse-aggregation-app-endpoint-fail-user-key.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+
+[id='calling-fuse-aggregation-app-endpoint-fail-user-key_{context}']
+= calling-fuse-aggregation-app-endpoint-fail-user-key 
+
+. From the *ActiveDocs* page for the `{fuse-aggregator-app-name}` Application, expand the *GET /flights* endpoint
```
Comment (david-martin) [inline docs/modules/ROOT/pages/_partials/calling-fuse-aggregation-app-endpoint-fail-user-key.adoc]: Is it necessary to have this line each time?
Would it be enough just the first time as the user would still be on the same page with the Flights API already open?

**Comment by david-martin** [comment]
@aidenkeating I can't approve as I opened the PR originally.
But latest review and veirfication looks good.
I'd suggest merging as is, with follow up testing to surface any further issues

**Pull request merged**